library/src/main/java/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -200,17 +203,15 @@ public FluentInitializer init(Date minDate, Date maxDate, TimeZone timeZone, Loc
     minCal = Calendar.getInstance(timeZone, locale);
     maxCal = Calendar.getInstance(timeZone, locale);
     monthCounter = Calendar.getInstance(timeZone, locale);
-    monthNameFormat =
-        new SimpleDateFormat(getContext().getString(R.string.month_name_format), locale);
-    monthNameFormat.setTimeZone(timeZone);
     for (MonthDescriptor month : months) {
-      month.setLabel(monthNameFormat.format(month.getDate()));
+      month.setLabel(formatMonthDate(month.getDate()));
     }
     weekdayNameFormat =
         new SimpleDateFormat(getContext().getString(R.string.day_name_format), locale);
     weekdayNameFormat.setTimeZone(timeZone);
     fullDateFormat = DateFormat.getDateInstance(DateFormat.MEDIUM, locale);
     fullDateFormat.setTimeZone(timeZone);
+    monthFormatter = new Formatter(monthBuilder, locale);
```

Comment:
redundant? assigned above.

Reply:
Yeap! I am changing here! Thanks for reviewing!